### ISSUE TITLE
Deploy bosh-deployment(bosh-lite) v2 error.

### ISSUE THREAD

**Issue by vnmaienso** [cloudfoundry/bosh-lite#435]
Hi All,
I am trying to deploy bosh-deployment v2 on bosh-lite following as https://bosh.io/docs/bosh-lite.html link.
I create Ubuntu 14.04 LTS on virtual box to deploy bosh-deployment. When i run command as below:
$ bosh create-env ~/workspace/bosh-deployment/bosh.yml \
  --state ./state.json \
  -o ~/workspace/bosh-deployment/virtualbox/cpi.yml \
  -o ~/workspace/bosh-deployment/virtualbox/outbound-network.yml \
  -o ~/workspace/bosh-deployment/bosh-lite.yml \
  -o ~/workspace/bosh-deployment/bosh-lite-runc.yml \
  -o ~/workspace/bosh-deployment/jumpbox-user.yml \
  --vars-store ./creds.yml \
  -v director_name="Bosh Lite Director" \
  -v internal_ip=192.168.50.6 \
  -v internal_gw=192.168.50.1 \
  -v internal_cidr=192.168.50.0/24 \
  -v outbound_network_name=NatNetwork

an error display:

Started installing CPI
  Compiling package 'golang_1.7/21609f611781e8586e713cfd7ceb389cee429c5a'... Finished (00:00:23)
  Compiling package 'virtualbox_cpi/e293cbbb8359fd2cbbb9777b7b91fd142ab6c688'... Finished (00:00:17)
  Installing packages... Finished (00:00:02)
  Rendering job templates... Failed (00:00:00)
Failed installing CPI (00:00:43)

Started deploying
  Creating VM for instance 'bosh/0' from stemcell 'sc-30fe9540-428f-48da-7949-163bc2388fa5'... Failed (00:01:09)
Failed deploying (00:01:09)

Stopping registry... Finished (00:00:00)
Cleaning up rendered CPI jobs... Finished (00:00:00)

Deploying:
  Creating instance 'bosh/0':
    Creating VM:
      Creating vm with stemcell cid 'sc-30fe9540-428f-48da-7949-163bc2388fa5':
        CPI 'create_vm' method responded with error: CmdError{"type":"Bosh::Clouds::CloudError","message":"Creating VM with agent ID '{{e8468f0a-1ace-45e9-73ca-bcc21aadd74b}}': Starting VM: Retried '30' times: Running command: 'VBoxManage startvm vm-004f1683-b265-4484-69b9-d694e3cc105d --type headless', stdout: 'Waiting for VM \"vm-004f1683-b265-4484-69b9-d694e3cc105d\" to power on...\n', stderr: 'VBoxManage: error: VT-x is not available (VERR_VMX_NO_VMX)\nVBoxManage: error: Details: code NS_ERROR_FAILURE (0x80004005), component ConsoleWrap, interface IConsole\n': exit status 1","ok_to_retry":false}

Exit code 1

Please help me resolve this error if you know about it.

Thank so much.

**Comment by cppforlife** [comment]
i think you might have missed this step for cli v2: https://bosh.io/docs/cli-env-deps.html

**Comment by vnmaienso** [comment]
Hi @cppforlite,
Thanks for your response.
I have resolve this error.

Thank s much.

**Comment by tezizzm** [comment]
Hi vnmaienso,
I am going through the same tutorial and have run into the same problem you have.  I was hoping you would share how you resolved this error to help me and anybody else who may stumble into the same issue.  Thanks.

**Comment by pivotal-jamil-shamy** [comment]
@tezizzm are you running `create-env` inside a virtualbox created VM?

**Comment by tezizzm** [comment]
Hi @pivotal-jamil-shamy I was able to solve my issue.  My host machine is a Windows Machine.  I have a client VM running Linux and from there I was calling `create-env `.  I'll see if I can find the link but VirtualBox doesn't pass the VT-x setting through from the host to the client which explains the error "VT-x is not available".  As a workaround I exported the VirtualBox VM to VMWare Workstation Player which is capable of passing the VT-x setting from the host to the client and everything worked well.
